lua/autorun/webaudio.lua

```diff
@@ -483,19 +509,21 @@ local function loadWhitelist(reloading)
 		end
 
 		notify("Whitelist from webaudio_whitelist.txt found and parsed with %d entries!", ind)
-		Whitelist = new_list
+		if SERVER then
+			Whitelist = new_list
+			WebAudio.Common.Whitelist = new_list
+			WebAudioStatic.sendCustomWhitelist(Whitelist)
```

Comment:
This should be WebAudio.sendCustomWhitelist(...) instead of directly accessing the static table

Reply:
Done